examples/index.html

```diff
@@ -163,13 +163,13 @@ <h3>Directly access to <span class="text-normal text-italic">bpmn-visualization<
                     <h5>Content</h5>
                     <ul>
                         <li><a href="#demos">Demos</a></li>
-                        <li><a href="#display-bpmn-theme">Display BPMN Diagram examples</a></li>
-                        <li><a href="#diagram-navigation">Diagram navigation examples</a></li>
-                        <li><a href="#overlays">Overlay examples</a></li>
-                        <li><a href="#custom-bpmn-theme">Custom BPMN Theme examples</a></li>
-                        <li><a href="#custom-animation">Custom animation examples</a></li>
-                        <li><a href="#custom-behavior">Custom behavior examples</a></li>
-                        <li><a href="#miscellaneous">Miscellaneous examples</a></li>
+                        <li><a href="#display-bpmn-theme">BPMN Diagram rendering</a></li>
+                        <li><a href="#diagram-navigation">BPMN Diagram navigation</a></li>
```

Comment:
We choose the wording "**Diagram navigation**" in our last conversation.

```suggestion
                        <li><a href="#diagram-navigation">Diagram navigation</a></li>
```